subsys/bluetooth/controller/ll_sw/ull_connections.c

```diff
@@ -506,6 +516,16 @@ int ull_conn_llcp(struct ll_conn *conn, uint32_t ticks_at_expire, uint16_t lazy)
 {
 	LL_ASSERT(conn->lll.handle != ULL_HANDLE_NOT_CONNECTED);
 
+#if defined(CONFIG_BT_CTLR_CONN_PARAM_REQ)
```

Comment:
I realized this is needed re. procedures that use instant (I had missed it re. phy update), so it should be passed in llcp and not llcp.conn_upd

Reply:
Will move to llcp.